src/main/java/core/basesyntax/service/impl/WriterService.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.Writer;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriterService implements Writer {
+    private static final String DIVIDER = ",";
+    private static final String HEADER = "fruit,quantity";
+    private final String outputFIle;
+    private final List<Fruit> fruitData;
+
+    public WriterService(String outputFIle, List<Fruit> fruitData) {
```

Comment:
let's pass data as `String report, String outputFilePath` into method parameters, and not into constructor

this will allow us to reuse this method not only for writing info about fruits, but any data